deployment/src/main/java/io/quarkiverse/loggingui/quarkus/logging/ui/deployment/LoggingUiProcessor.java

```diff
@@ -36,40 +57,142 @@ FeatureBuildItem feature() {
 
     @Record(ExecutionTime.STATIC_INIT)
     @BuildStep
-    void buildExecutionService(BuildProducer<RouteBuildItem> routeProducer,
+    void incluceRestEndpoints(BuildProducer<RouteBuildItem> routeProducer,
```

Comment:
Typo: should it be `incluDeRestEndpoints`?

Reply:
Thanks ! Fixed